src/jit/compiler.cpp

```diff
@@ -4815,8 +4815,8 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, JitFlags
     m_pLinearScan = getLinearScanAllocator(this);
 
     /* Lower */
-    Lowering lower(this, m_pLinearScan); // PHASE_LOWERING
-    lower.Run();
+    m_pLowering = new (this, CMK_LSRA) Lowering(this, m_pLinearScan); // PHASE_LOWERING
```

Comment:
Ah, I see, lowering ends up being used by `fgExtendDbgLifetimes`, that's a bit odd.